cloud_launch_large.json

```diff
@@ -1,13 +1,13 @@
 {
-  "template": "medium_small",
+  "template": "medium_small_entity_db_v2",
   "world": {
     "chunkEdgeLengthMeters": 50,
     "snapshots": {
       "snapshotWritePeriodSeconds": 0
     },
     "dimensions": {
-      "xMeters": 1600,
-      "zMeters": 1600
+      "xMeters": 900,
+      "zMeters": 900
```

Comment:
Any reason for this change?

Reply:
Large world has 12 layers ==> (12*2*36 + 4) ==> 868m edge length 

so the spatial world only needs to be 900x900